MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -75,6 +75,56 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
             loadRequestedAd(with: callbackId, for: requestParam)
         case .showAd:
             showRequestedAd(with: callbackId, for: requestParam)
+        case .sendMessageToContact:
+            sendMessageToContact(with: callbackId, parameters: requestParam)
+        case .sendMessageToContactId:
+            sendMessageToContact(with: callbackId, parameters: requestParam)
+        case .sendMessageToMultipleContacts:
+            sendMessageToMultipleContacts(with: callbackId, parameters: requestParam)
+        }
+    }
+
+    private func sendMessageToContact(with callBackId: String, parameters: RequestParameters?) {
+        if let message = parameters?.messageToContact {
+            if let contactId = parameters?.contactId {
+                hostAppMessageDelegate?.sendMessageToContact(contactId, message: message) { result in
+                    switch result {
+                    case .success:
+                        self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callBackId, response: "")
+                    case .failure(let error):
+                        self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: error.localizedDescription)
+                    }
+                }
+            } else {
+                hostAppMessageDelegate?.sendMessageToContact(message, completionHandler: { result in
+                    switch result {
+                    case .success(let contactId):
+                        self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callBackId, response: contactId ?? "")
+                    case .failure(let error):
+                        self.executeJavaScriptCallback(responseStatus: .onError, messageId: callBackId, response: error.localizedDescription)
+                    }
+                })
+            }
+        }
+    }
```

Comment:
I used the same method for `sendMessageToContact` and `sendMessageToContactId` because the 2 payloads can be differentiated by the presence of `contactId`

Reply:
I think it might make more sense to use `sendMessageToContactId` so that it matches the JavaScript function name.

Reply:
Understood